hack/setup-roles.sh

```diff
@@ -0,0 +1,14 @@
+SCRIPT_DIR=website/content/en/preview/getting-started/getting-started-with-eksctl/scripts
+TEMP_DIR=$(mktemp -d)
```

Comment:
Do you even need the temp dir, since you can just execute these straight out of the file system?